Bottom: b56234e86a7b171435d4772b24d073ead64be41c
Top:    a52548fef076470c90587d2328e8174b473a229d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 03:41:46 +0000

Turn off debug hacks in object generation


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 0723343..3468aec 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1483,9 +1483,9 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   destroy_appearance (copper_appearance);
 
   /* DEBUG */
-  poly_M_Copy0 (&PCB->Data->outline, info.poly);
-  PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  poly_M_Copy0 (&PCB->Data->outline, info.poly);
+//  PCB->Data->outline_valid = true;
+//  gui->invalidate_all ();
 
   poly_Free (&info.poly);
